numba/cuda/compiler.py

```diff
@@ -283,6 +283,10 @@ def compile_ptx(pyfunc, sig, debug=False, lineinfo=False, device=False,
                 ``"c"``. Note that the Numba ABI is not considered stable.
                 The C ABI is only supported for device functions at present.
     :type abi: str
+    :param abi_info: A dict of ABI-specific options. The ``"c"`` ABI supports
+                     one option, ``"abi_name"``, for providing the wrapper
+                     function's name. The ``"numba"`` ABI has no options.
```

Comment:
Should the code assert that if the ABI in use is `"numba"` then the `abi_name` key in the `abi_info` dictionary is not set?

Reply:
I think it would be a strange precedent to set - what if another ABI adds another option - should all ABIs assert that they are only passed specifically the items they support? (My leaning is that they should not, simply because the burden of doing that guarding outweighs the potential small benefit of occasionally catching passing an irrelevant / misspelled key)